build_docs.pl

```diff
@@ -489,6 +489,7 @@ sub init_repos {
         user      => $Opts->{user},
         url       => $Opts->{target_repo},
         reference => $reference_dir,
+        keep_hash => 0,
```

Comment:
You can't `keep_hash` on the target repo because the `target_repo` contains the list of hashes to keep!

Reply:
If it's worth a review comment, is it worth a code comment?